### DISCUSSION TITLE
Looking for New Maintainers!

### DISCUSSION THREAD

**Discussion by stokkie90** [UI-Lovelace-Minimalist/UI#1560] [General]
This project is in need of new maintainers! The current maintainers are no longer actively working on it, and without fresh leadership, we may be forced to shut down the project and its associated Discord server.

If you're passionate about this project and would like to take over its development and community management, we’d love to hear from you!

Thank you for considering helping keep this project alive and thriving! 🙌

**Comment by warmfire540** [comment]
Where can I find out more information?

**Reply by stokkie90** [reply]
Any specifics you would like to know?

**Reply by warmfire540** [reply]
what are requirements / expectations?  I'm glad to help w/ PRs and helping with issues, ci/cd, hacs and such..

I wasn't able to click the discord link, it goes to a dead link for me

**Reply by KruseLuds** [reply]
https://github.com/UI-Lovelace-Minimalist/UI

**Reply by warmfire540** [reply]
@KruseLuds you sent repo link, was looking for info about maintainers

**Reply by warmfire540** [reply]
Is there an updated discord link? The one in the readme isn’t working for me

**Reply by johnsonbui** [reply]
@warmfire540 Did you ever figure it out? Kinda curious myself.

**Comment by jmorton05** [comment]
If you are in need of additional help I am interested. I don't generally use discord... but could be persuaded.
